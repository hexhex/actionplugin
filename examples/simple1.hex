b(u).
b(v).
c(u).
d(v).
c(e).
d(e).
a(X) :- b(X), c(X).
a(X) :- b(X), d(X).
a(X) :- c(X), d(X).
