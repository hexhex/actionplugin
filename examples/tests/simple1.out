{a(e), a(v), a(u), d(e), d(v), c(e), c(u), b(v), b(u)}
